src/components/profiles/Officials.tsx

```diff
@@ -24,24 +25,24 @@ interface OfficialsData {
     }
 }
 
-const OfficialCard: React.FC<{lang: string, official: Member}> = ({lang, official}) => {
+const OfficialCard: React.FC<{lang: string, official: Member, image: DynamicImageData}> = ({lang, official, image}) => {
     const leader = official.leader ? style.leader : ''
     return (
         <div className={`${style.official_container} ${leader}`} >
-            <ProfileImg src={official.picture}/>
+            <DynamicProfileImg data={image} alt={official.name} />
             <div className={style.name}>{official.name}</div>
             {official.title.map(title => <div key={title[lang]} className={style.title}>{title[lang]}</div>)}
         </div>
     )
 }
 
 const CommitteeSection: React.FC<{lang: string, committee: Committee}> = ({lang, committee}) => {
-    const {name, members}: {name: TranslatedEntry, members: Member[]} = committee.node;
+    const {name, members, localImage}: {name: TranslatedEntry, members: Member[], localImage: DynamicImageData[]} = committee.node;
     return (
         <div className={style.officials_section}>
             <h2 id="name">{name[lang]}</h2>
             <div className={style.officials_list}>
-                {members.map(official => <OfficialCard key={official.name} lang={lang} official={official} />)}
+                {members.map((official, i) => <OfficialCard key={official.name} lang={lang} image={localImage[i]} official={official} />)}
```

Comment:
Does this mean the image object is not actually tied to the rest of the data and we just hope the two are in sync?